HACKING.md

```diff
@@ -131,6 +131,19 @@ assuming they have enough permissions.
 oc create secret generic gcp-kola-tests-config --from-file=config=/path/to/kola-secret
 ```
 
+### [OPTIONAL] Creating OpenStack credentials configs
+
+If you want to run kola tests against OpenStack images you need to
+create a secret (`clouds.yaml` format) for talking with the openstack
```

Comment:
Minor/optional: is there a page we could link to which describes `clouds.yaml`?

Also

```suggestion
create a secret (`clouds.yaml` format) for talking with the OpenStack
```

:)

Reply:
Yep will link.